flocker/node/test/test_loop.py

```diff
@@ -350,7 +350,7 @@ def test_convergence_done_unchanged_notify(self):
         loop = build_convergence_loop_fsm(reactor, deployer)
         loop.receive(_ClientStatusUpdate(
             client=client, configuration=configuration, state=state))
-        reactor.advance(1.0)
+        reactor.advance(ConvergenceLoop._UNCONVERGED_DELAY)
```

Comment:
Good point. Not feeling motivated enough to fix this though since it doesn't leak out into public API.